app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name}"
+            "(first_name, last_name) VALUES (?, ?) ",
+            (first_name, last_name,)
+        )
+        self._connection.commit()
+
+    def all(self) -> None:
+        actors_data = self._connection.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"

```

Comment:
Use `select *`, because if you will change Actor model you also need change here